build.gradle

```diff
@@ -91,7 +91,7 @@ if (!usingRemoteCluster) {
 }
 
 ext {
-    opendistroVersion = '1.0.0'
+    opensearchVersion = '1.0'
```

Comment:
Please continue to use 1.0.0.0 for the plugin version. 
ie. `opensearchVersion = '1.0.0'`